---
layout: default
title: Projects
nav_order: 1
---

## Microbiology Projects

- **Omics analysis and modeling of wetland communities**  
  
  The wetland ecosystems are responsible for one third of global emission of methane, a potent greenhouse gas. The methane emission from wetlands will be 
  perturbed by climate changes, including seawater intrusion and frequent droughts. In this project, we aim to characterize and model the ecosystem-scale 
  biogeochemical fluxes in estuarine wetlands and lacustrine wetlands under climate changes. Our approaches include proteomic stable isotope probing, 
  meta-omics analysis, and computational modeling. Our deliverables are predictive models of wetland communities to inform global Earth system models.  
  
  Lab members: Dongyu Wang, Yi Xiong, Chongle Pan.  
  Collaborators: Dr. [Mari Winkler](http://www.winklerlab.com) at UW Seattle.  
  Sponsor: Department of Energy, Biological and Environmental Research.  

- **Prebiotics and protiobics in humane microbiomes**

  Human microbiomes play important roles in our health. While prebiotics and probiotics have been shown to improve many health outcomes, their mechanisms are
  not known. In this project, we aim to link the metabolisms of prebiotics with specific microorganisms in the gut microbiomes and characterize their interaction
  mechanisms. Our discoveries will enable the development of prebiotics and probiotics with enhanced health benefits. 

  Lab members: Stormi Miller, Yi Xiong, Chongle Pan.  
  Collaborators: Dr. [Ryan Mueller](https://microbiology.oregonstate.edu/dr-ryan-mueller) at Oregon State Univ.  
  Sponsor: National Institutes of Health.  

## Computer Science Projects

- **Explainable machine learning for predictive genomics**

  Many complex diseases, including cancers and diabetes, are caused by a combination of genetic factors and environmental factors. The genetic risk of an individual
  for a complex disease is determined by their genome. In this project, we aim to improve the prediction of complex disease occurance in individuals from their 
  whole genome using machine learning. Explainable neural network models are developed to ensure their predictions are trustworthy. Our model predictions will 
  enable precision preventive medicine tailored specifically for individuals based on their disease risks. 
  
  Lab members: Adrien Badré, Justin Reynolds, Chongle Pan.  
  
- **Computer vision for biomedical imaging**

  Deep learning enables automatic analysis of many imaging modalities for disease diagnoses and surgery intervention. Endoscopic optical coherence tomography can
  be used to guide the needle placement in many needle-based surguries. In this project, we aim to develop computer vision models for automatic interpretation of 
  optical coherence tomography imaging data. Precise needle placement will improve the safety of needle-based surguries, such as epidural anesthesia and 
  percutaneous nephrostomy.

  Lab members: Paul Calle Contreras, Justin Reynolds, Chongle Pan.  
  Collaborator: Dr. [Qinggong Tang](http://tanglab.oucreate.com/) at OU

- **Natural text understanding in biomedical informatics** 

  Mobile technologies, such as smartphones and smart watches, allows real-time health monitoring and care provision. In this project, we aim to develop predictive
  models based on mobile health data and to develop chatbots for real-time intervention. We are developing a smoke-cessation chatbot may provide basic counseling 
  at critical moments and help users resist craving. 
  
  Lab members: Yunlong Liu, Adrien Badré, Chongle Pan.  
  Collaborators: Dr. [Michael Businelle](http://www.businellelab.com) at OU HSC. 

- **Parallel computing for proteomic stable isotope probing**

  Proteomic stable isotope probing are developed in our lab to track the flow of carbon and nitrogen from a substrate to its consumers in a complex community.
  In this project, we aim to improve Sipros to achieve more accurate detection of isotopically labeled peptides from shotgun proteomics data. Computing will be
  accelerated with GPU and provided in the clound. This will make proteomic stable isotope probing more accessible for the microbiome community.

  Lab members: Yi Xiong, Dongyu Wang, Chongle Pan.  
  Collaborators: Dr. [Xuan Guo](http://www.cse.unt.edu/~xuanguo/) at UNT.  
  Sponsor: Department of Energy, Biological and Environmental Research

## Completed Projects
- High-performance bioinformatics workflow for integrative ‘-omics’ data analytics (2016 ~ 2019) funded by DOD
- Genomic and functional characterization of the microbiota of obese African Americans (2017 ~ 2019) funded by NIH
- A quantitative, systems biology, multi-omic approach to diagnose and predict response to treatment for gynecologic cancers (2017 ~ 2019) funded by ORNL
- A platform to identify and model genetic and phenotypic changes in cancer at the single cell level (2017 ~ 2019) funded by ORNL
- Multi-‘omic’ analyses of the dynamics, mechanisms, and pathways for carbon turnover in grassland soil under two climate regimes (2013 ~ 2018) funded by DOE BER
- Characterization of diazotrophs in the endosphere microbiome of bioenergy crop Sorghum (2016 ~ 2018) funded by ORNL
- Individual diploid genome sequencing with parental haploid resolution and structural variation identification (2015 ~ 2016) funded by ORNL
- Predicting climate feedbacks from microbial function in tropical ecosystems (2014 ~ 2016) funded by ORNL
- Systems level dissection of anaerobic methane cycling (2013 ~ 2016) funded by DOE BER
- Development of a pipeline for high-throughput recovery of near-complete and complete microbial genomes from complex metagenomic datasets (2013 ~ 2015) funded by JGI
- Linking phylogenetic identity and biogeochemical function of uncultivated marine microbes with novel mass spectrometry techniques (2012 ~ 2015) funded by Moore Foundation
- Improved metagenomic analysis with confidence quantification for biosurveillance of novel and man-made threats (2012 ~ 2014) funded by ORNL
- Development and integration of genome-enabled techniques to track and predict the cycling of carbon in model microbial communities (2010 ~ 2013) funded by DOE BER
- Development of a data integration system for microbial community systems biology data (2010 ~ 2013) funded by DOE BER
- Syntrophic Interactions and Mechanisms Underpinning Anaerobic Methane Oxidation (2010 ~ 2013) funded by DOE BER
- Plant-microbe interfaces (2009 ~ 2018) funded by DOE BER
- Ultrascale computational modeling of phenotype-specific metabolic processes in microbial communities (2008 ~ 2011) funded by DOE ASCR

